### PR TITLE
Move examples to separate files

### PR DESCRIPTION
**Pull request by bdFregat**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  |  Move examples to separate files #14

### CODE REVIEW

File: docs/composer.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+### Using [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
```
Comment (vjik) [inline docs/composer.md:1]:
```suggestion
# Using [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
```

And rename to `composer-require-checker.md`

File: docs/php-bench.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+### Example of use of the action [PHPBench](https://github.com/phpbench/phpbench)
```
Comment (vjik) [inline docs/php-bench.md:1]:
```suggestion
# Using [PHPBench](https://github.com/phpbench/phpbench)
```

And rename to `phpbench.md`

File: docs/php-unit.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+### Example of use of the action [PHPUnit](https://github.com/sebastianbergmann/phpunit)
```
Comment (vjik) [inline docs/php-unit.md:1]:
```suggestion
# Using [PHPUnit](https://github.com/sebastianbergmann/phpunit)
```

And rename to `phpunit.md`

File: docs/psalm.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+### Example of use of the action [PSALM](https://github.com/vimeo/psalm)
```
Comment (vjik) [inline docs/psalm.md:1]:
```suggestion
# Using [Psalm](https://github.com/vimeo/psalm)
```

File: docs/rector.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+### Example of use of the action [RECTOR](https://github.com/rectorphp/rector)
```
Comment (vjik) [inline docs/rector.md:1]:
```suggestion
### Using [Rector](https://github.com/rectorphp/rector)
```

File: docs/roave-infection.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+### Example of use of the action [ROAVE-INFECTION](https://github.com/roave/infection-static-analysis-plugin)
```
Comment (vjik) [inline docs/roave-infection.md:1]:
```suggestion
# Using [Infection](https://infection.github.io/) mutation framework with [Infection Static Analysis Plugin](https://github.com/Roave/infection-static-analysis-plugin)
```

File: README.md [hunk 7]
```diff
@@ -11,230 +11,13 @@
...
-    uses: yiisoft/actions/.github/workflows/roave-infection.yml@master
-    with:
-      os: >-
-        ['ubuntu-latest']
-      php: >-
-        ['8.1']
-    secrets:
-      STRYKER_DASHBOARD_API_KEY: ${{ secrets.STRYKER_DASHBOARD_API_KEY }}
-```
+## Examples:
```
Comment (vjik) [inline README.md:14]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -11,230 +11,13 @@
...
+- [Composer require checker](docs/composer.md)
+- [PHPBench](docs/php-bench.md)
+- [PHPUnit](docs/php-unit.md)
+- [PSALM](docs/psalm.md)
```
Comment (vjik) [inline README.md:18]:
```suggestion
- [Psalm](docs/psalm.md)
```

File: README.md [hunk 9]
```diff
@@ -11,230 +11,13 @@
...
+- [Rector](docs/rector.md)
+- [Roave Infection](docs/roave-infection.md)
```
Comment (vjik) [inline README.md:20]:
```suggestion
- [Infection with Static Analysis Plugin](docs/roave-infection.md)
```

**Review by vjik — CHANGES_REQUESTED** [review]

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Thanks!

**Pull request merged**